### ISSUE TITLE
Bikes

### ISSUE THREAD

**Issue by hitzu111** [StrangeLoopGames/EcoSuggestions#1264]
With the addition of labor and labor workers sometimes there is little need to carry heavy stuff but fast travel on long distances is desirable. Some compact and fast means of transportation such as bicycles and motorbikes will be a nice way to travel around.

**Comment by ThePiachu** [comment]
Yeah, I had the same idea recently. Making it like a cheap Machinist recipe would be interesting. Make it light enough to pick up without an issue (lighter than a Cart), give it a small storage slot for when you need to do a small courier run and it could fill a cool niche of light transport.

**Comment by hitzu111** [comment]
Yeah. The first bicycles may be in a tech tree before the Steam Cart. They may even replace the Steam Cart and could have modules just like a tractor has: 

- a module for larger storage space along with a third wheel for support

- this module could have an alternative version for a second seat

- a module for a steam motor and solid fuel slots

- a module for a combustion engine and liquid fuel slots

Basic Engineers will make a basic version out of Iron Wheels, Iron Pipes, and Boards. A larger compartment module will be made out of Cloth, an Iron Wheel, Iron pipes, and Boards. A second seat will be the same but with Leather Hide in its recipe. And the motor modules will be made of the Boiler and Combustion Engine respectively.

This way players could upgrade their old vehicles as a cheaper alternative for heavier trucks with a trade-off for lesser capacity.
